docs/development_guide.rst

```diff
@@ -3,27 +3,60 @@ Development Guide
 
         *----------------------------- Content in Development -----------------------------*
 
-Development Setup
------------------
+Example RackHD Setup
+--------------------
 
 **Prerequisites**
 
-Install Node.js and npm. **Note:** Currently, node versions 0.10.x are supported. node 0.12 and node 4.0 will be supported in the near future.
+The example setup uses Vagrant and ansible locally. Please install them
+directly on your local machine.
+
+
+
+1. Clone at minimum the rackHD repository. This repository includes both git
+   submodules with all the relevant source code, as well as an example vagrant
+   instance to get an instance up and running locally very easily.
+
+   `git clone https://github.com/RackHD/RackHD`
+
+   Within the example/ directory is a setup script to set up an "example rack"
+   entirely virtually - albiet a very simple one. It creates a virtualbox with
+   RackHD running within it, and another that is connected to it to PXE boot
+
+2. Create a local configuration file and spin up vagrant
+
+   `cd RackHD`
+   `cp example/config/monorail_rack.cfg.example example/config/monorail_rack.cfg`
+
+3. Fire up the virtual rack
+
+   `cd example/bin`
+   `./monorail_rack`
 
+4. ssh into the RackHD server and use on-imagebuilder to build the static files
 
+   `vagrant ssh dev`
+   `sudo apt-get install -y ansible`
+   `cd ~`
+   `git clone https://github.com/rackhd/on-imagebuilder`
+   `cd on-imagebuilder`
+   `sudo ansible-playbook -i hosts all.yml`
 
-**Checking Out the Code**
+5. Move the resulting files into the HTTP server's static files directory
 
-1. Clone at minimum the **on-http**, **on-dhcp-proxy**, **on-tftp**, **on-taskgraph**, and **on-tasks** repositories.
+   `mv /tmp/on-imagebuilder/builds/* ~/RackHD/on-http/static/http/common`
```

Comment:
As @DavidjohnBlodgett pointed out in the other PR, change the second path to `~/src/on-http/static/http/common/`